src/Entity/CommitteeMembership.php

```diff
@@ -85,6 +90,15 @@ class CommitteeMembership
      */
     private $joinedAt;
 
+    /**
+     * Indicates if the adherent votes in this committee
+     *
+     * @var bool
+     *
+     * @ORM\Column(type="boolean")
```

Comment:
What about default value? `, options={"default": false})` (need to change migration)

Reply:
I didn't know if you'd prefer app-managed or db-managed defaults, changing!